panacea/query_client.go

```diff
@@ -48,14 +48,16 @@ type QueryClient interface {
 	GetDeal(context.Context, uint64) (*datadealtypes.Deal, error)
 	GetConsent(context.Context, uint64, string) (*datadealtypes.Consent, error)
 	GetLastBlockHeight(context.Context) (int64, error)
+	GetCachedLastBlockHeight() int64
 	GetOracleUpgrade(context.Context, string, string) (*oracletypes.OracleUpgrade, error)
 	GetOracleUpgradeInfo(context.Context) (*oracletypes.OracleUpgradeInfo, error)
 	GetOracle(context.Context, string) (*oracletypes.Oracle, error)
 	VerifyTrustedBlockInfo(int64, []byte) error
 }
 
 const (
-	trustedPeriod = 2 * 365 * 24 * time.Hour
+	trustedPeriod       = 2 * 365 * 24 * time.Hour
+	refreshIntervalTime = time.Second * 3
```

Comment:
This interval seems not to be used.

Reply:
Thank you.
I forgot to use this value.
Fixed(https://github.com/medibloc/panacea-oracle/pull/101/commits/a1abb012f74e92c8f24cf2d234873eb64d2fb094)